src/OneOffPayment/OneOffPaymentBuilder.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+namespace Laravel\Cashier\OneOffPayment;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Str;
+use Laravel\Cashier\Cashier;
+use Laravel\Cashier\Mollie\Contracts\CreateMolliePayment;
+use Laravel\Cashier\Mollie\Contracts\UpdateMolliePayment;
+use Laravel\Cashier\Order\OrderItemCollection;
+use Laravel\Cashier\Payment;
+use Mollie\Api\Types\SequenceType;
+
+class OneOffPaymentBuilder
+{
+    /**
+     * The billable model.
+     *
+     * @var \Illuminate\Database\Eloquent\Model
+     */
+    protected $owner;
+
+    /**
+     * A collection of items that will be added to the Order
+     *
+     * @var OrderItemCollection
+     */
+    protected $items;
```

Comment:
How about we instead store an `AddGenericOrderItem` action for each item on the local Payment record when using `forItems`?

That way we can expose both the current `forItems()` method and an additional `forActions()` method. Both will be using the actions under the hood - allowing really cool custom integrations.

It also makes it consistent with the way the FirstPaymentBuilder handles order items.